src/main/java/org/assertj/core/api/AbstractLocalDateAssert.java

```diff
@@ -494,4 +527,15 @@ private static void assertLocalDateParameterIsNotNull(LocalDate other) {
     checkArgument(other != null, NULL_LOCAL_DATE_TIME_PARAMETER_MESSAGE);
   }
 
+  /**
+   * Check that the {@link XMLGregorianCalendar} to compare actual {@link LocalDate} to is not null, in that case throws
+   * a {@link IllegalArgumentException} with an explicit message
+   *
+   * @param xmlGregorianCalendar the {@link XMLGregorianCalendar} to check
+   * @throws IllegalArgumentException with an explicit message if the given {@link XMLGregorianCalendar} is null
+   */
+  private static void assertXMLGregorianCalendarParameterIsNotNull(XMLGregorianCalendar xmlGregorianCalendar) {
```

Comment:
Should be in `AbstractTemporalAssert` as there is nothing specific to `LocalDate` assertions.